app/models/protocol.rb

```diff
@@ -2,6 +2,7 @@ class Protocol < ApplicationRecord
   has_many :protocol_drugs, -> { order(:updated_at) }
 
   has_many :facility_groups
+  has_many :drug_stocks
```

Comment:
This should be on the protocol _drug_, right?

Reply:
Oh, whoops